test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -361,8 +361,8 @@ public async Task DependencyGraphRestoreUtility_LegacyPackageRef_Restore_ReadLoc
                     Assert.True(File.Exists(projectLockFilePath));
 
                     // delete existing restore output files
-                    File.Delete(Path.Combine(vsProjectAdapter.MSBuildProjectExtensionsPath, "project.assets.json"));
-                    File.Delete(Path.Combine(vsProjectAdapter.MSBuildProjectExtensionsPath, NoOpRestoreUtilities.NoOpCacheFileName));
+                    File.Delete(Path.Combine(await vsProjectAdapter.GetMSBuildProjectExtensionsPathAsync(), "project.assets.json"));
```

Comment:
Why are we calling the same async method to get path over and over? Can't this be called once, stored, and passed in to the dozen or so calls below?

Reply:
Sure it can, thanks for noticing!